components/track/TrackItem.vue

```diff
@@ -48,6 +48,33 @@ function adjustHighlightText(highlight: Highlighting) {
     .replaceAll("**/", "</span>");
 }
 
+const fields = computed(() => {
```

Comment:
Some of this logic is repeated in the `trackFields` function in `utils/trackFields.ts`.
Could we use the util here, or is the logic too custom here?

Reply:
TrackItem has some special logic. I thought it makes sense to keep the 2 separate.
But I'm happy the logic now is only in 2 places instead of 5.

Reply:
I also added another special case for when the album is known after your review.